packages/babel-plugin-transform-classes/src/transformClass.ts

```diff
@@ -452,11 +452,7 @@ export default function transformClass(
             return true;
           }
 
-          if (
-            parentPath.isLoop() ||
-            parentPath.isConditional() ||
-            parentPath.isArrowFunctionExpression()
```

Comment:
Out of curiosity, why can `isLoop` and `isArrowFunctionExpression` be safely removed?